metricbeat/module/windows/perfmon/pdh_query_windows.go

```diff
@@ -274,7 +274,7 @@ func UTF16ToStringArray(buf []uint16) []string {
 	stringLine := UTF16PtrToString(&buf[0])
 	for stringLine != "" {
 		strings = append(strings, stringLine)
-		nextLineStart += len(stringLine) + 1
+		nextLineStart += len([]rune(stringLine)) + 1
```

Comment:
Is it possible to add unit tests for this function/or this specific case?

Reply:
I have added a test for this